README.md

```diff
@@ -26,124 +23,65 @@ Inspired by [awesome-sar](https://github.com/RadarCODE/awesome-sar) with initial
 * [CryoSat-2](https://earth.esa.int/eogateway/missions/cryosat) - Radar altimetry mission offering complementary sea ice thickness and ice sheet elevation trends.
-* [Sentinel-1 SAR](https://sentinel.esa.int/) - Active microwave backscatter supporting surface type classification and change detection alongside elevation trends.
+* [Sentinel-1 SAR](https://dataspace.copernicus.eu/data-collections/sentinel-data/sentinel-1) - Active microwave backscatter supporting surface type classification and change detection alongside elevation trends.
 * [Landsat Collection](https://landsat.gsfc.nasa.gov/) - Multispectral imagery for land cover context and seasonal melt pond / vegetation mapping.
 * [MODIS / VIIRS Snow & Sea Ice](https://nsidc.org/data/modis) - Daily snow/ice products enabling temporal context for elevation/freeboard retrievals.
+* [DEM Auxiliary Data (REMA/ArcticDEM)](https://www.pgc.umn.edu/data/arcticdem/) - Polar DEM mosaics used for reference elevation and geolocation QA.
+* [ICESat GLAS (Legacy)](https://nsidc.org/data/icesat) - Historical laser altimetry providing multi-decadal context for elevation change.
+* [GRACE/GRACE-FO Mascon](https://podaac.jpl.nasa.gov/GRACE) - Complementary mass balance signals to compare with ICESat-2 elevation change in ice sheets.
+* [ERA5 Reanalysis](https://cds.climate.copernicus.eu/datasets) - Atmospheric state variables for correcting backscatter or interpreting surface processes.
 
 ## Community & Communication
 
-* [ICESat-2 Hackweek Slack](https://icesat-2.hackweek.io/) - Real-time discussion channels for workflows, troubleshooting, and collaboration (event-specific access).
 * [NSIDC User Support](https://nsidc.org/support) - Helpdesk for data access, documentation, and product interpretation questions.
 * [Earthdata Forum](https://forum.earthdata.nasa.gov/) - Official Q&A forum covering API access, subsetting, and dataset usage topics.
-* [AGU Cryosphere Sessions](https://www.agu.org/) - Conference sessions highlighting latest ICESat-2 science and methods developments.
-* [Twitter/X #ICESat2](https://twitter.com/search?q=%23ICESat2) - Social feed for mission news, data releases, and application highlights.
 
 ---
 
 ## Contributing
 
-Contributions welcome! Please open an issue or pull request adding a resource with: (1) Name, (2) URL, (3) one-sentence description, (4) category suggestion. Keep descriptions concise, neutral, and avoid promotional language. Duplicates or out-of-scope links (non-ICESat-2 focused) may be declined to maintain curation quality.
+Contributions welcome! Please open an issue or pull request adding a resource with: (1) Name, (2) URL, (3) one-sentence description, (4) (sub)category suggestion. Keep descriptions concise, neutral, and avoid promotional language. Duplicates or out-of-scope links (non-ICESat-2 focused) may be declined to maintain curation quality.
```

Comment:
Could be good to have further organization through at least one more subheading (`###`)